pyproject.toml

```diff
@@ -10,50 +10,49 @@ python = "~3.7"
 Django = "~3.0"
 djangorestframework = "~3.11"
 django-filter = "~2.2"
-pillow = "~7.0"
+pillow = "~7.1"
 markdown2 = "~2.3"
-pytz = "^2019.3"
+pytz = "^2020.1"
 python-memcached = "~1.59"
-stripe = "~2.37"
+stripe = "~2.48"
 icalendar = "~4.0"
 google-api-python-client = "~1.8"
 celery = "~4.4"
-PyMuPDF = "~1.16"
-django-filebrowser = { git = "https://github.com/sehmaschine/django-filebrowser.git", rev = "4c1a86d67cd7424df57b0e6880152b309823f009" }
-# ^ For Django 3 support
+PyMuPDF = "~1.17"
+django-filebrowser = "~3.13"
 django-chunks = { git = "https://github.com/dotKom/django-chunks.git", rev = "e6bf109d4fd286964024c140ac1b8f3d86713540" }
 django-crispy-forms = "^1.9"
 django-extensions = "^2.2"
 django-dynamic-fixture = "^3.1"
 django-recaptcha = "~2.0"           # Google ReCaptcha
-django-oauth-toolkit = "~1.2"       # OAuth2 authentication support
+django-oauth-toolkit = "~1.3"       # OAuth2 authentication support
 django-watson = "~1.5.5"            # Indexed model search lib
 django-reversion = "~3.0"           # Model version history with middleware hooks to all post_save signals
 # django-guardian = "~2.2"            # Per Object permissions framework
 django-guardian = { git = "https://github.com/django-guardian/django-guardian.git", rev = "dba092f5db86808713433388717dcfbaaa928a0b" }
 # ^ TODO: Update when django-guardian > 2.2 is released with fix for `get_objects_for_user` ref. dotkom/onlineweb4#2459
-django-taggit = "~1.2"              # Generic multi-model tagging library
+django-taggit = "~1.3"              # Generic multi-model tagging library
 django-taggit-serializer = "~0.1"   # REST Framework serializers for Django-taggit
 APScheduler = "~3.6"                # Scheduler
 redwine = {git = "https://github.com/dotkom/redwine.git", rev = "1.3.2"}
 reportlab = "~3.5"
-pdfdocument = "~3.3"
+pdfdocument = "~4.0"
```

Comment:
They changed the default font size from 8 to 9, which _might_ change the size of the text is some tables we are creating

Reply:
I think it will probably be OK